docs/cruisecontroloperation.md

```diff
@@ -5,120 +5,124 @@ weight: 200
 
-The process of the operation can be followed up through the `KafkaCluster` custom resource's status and through the created `CruiseControlOperation` custom resource's status.  
-In the section below you can see an example for the `add_broker` (`GracefulUpscale*`) operation but the same applies for the Kafka cluster `remove_broker` (`GracefulDownScale*`) and `rebalance` (when the `volumeState` is `GracefulDiskRebalance*`) operations.
+Cruise Control can execute only one task at a time, so the priority of the tasks depends on the type of the operation:
+
+- Upscale operations are executed first, then
+- downscale operations, then
+- rebalance operations.
+
+The following Cruise Control tasks are supported:
+
+- [add_broker](https://github.com/linkedin/cruise-control/wiki/REST-APIs#add-a-list-of-new-brokers-to-kafka-cluster) (`GracefulUpscale`)
+- [remove_broker](https://github.com/linkedin/cruise-control/wiki/REST-APIs#decommission-a-list-of-brokers-from-the-kafka-cluster) (`GracefulDownscale`)
+- [rebalance](https://github.com/linkedin/cruise-control/wiki/REST-APIs#trigger-a-workload-balance) (`GracefulDiskRebalance`)
+
+You can follow the progress of the operation through the `KafkaCluster` custom resource's status and through the `CruiseControlOperation` custom resource's status.  
+The following example shows the steps of an `add_broker` (`GracefulUpscale*`) operation, but the same applies for the Kafka cluster `remove_broker` (`GracefulDownScale*`) and `rebalance` (when the `volumeState` is `GracefulDiskRebalance*`) operations.
 
 1. Upscale the Kafka cluster by adding a new broker with id "3" into the `KafkaCluster` CR:
 
-```yaml
- Spec:
- ...  
-   brokers:
-    - id: 0
-      brokerConfigGroup: "default"
-    - id: 1
-      brokerConfigGroup: "default"
-    - id: 2
-      brokerConfigGroup: "default"
-    - id: 3
-      brokerConfigGroup: "default"
-...
-```
+    ```yaml
+    spec:
+    ...  
+      brokers:
+        - id: 0
+          brokerConfigGroup: "default"
+        - id: 1
+          brokerConfigGroup: "default"
+        - id: 2
+          brokerConfigGroup: "default"
+        - id: 3
+          brokerConfigGroup: "default"
+    ...
+    ```
 
-2. A new broker pod will be created and in the `KafkaCluster` status there will be the `cruiseControlOperationReference`.  
+1. A new broker pod is created and the `cruiseControlOperationReference` is added to the `KafkaCluster` status.  
    This is the reference of the created `CruiseControlOperation` custom resource.  
-   The `cruiseControlState` shows the `CruiseControlOperation` state.  
-   At first there will be `GracefulUpscaleScheduled`.  
-   This means `CruiseControlOperation` is created and it is waiting for `add_broker` task execution.
+   The `cruiseControlState` shows the `CruiseControlOperation` state: `GracefulUpscaleScheduled`, meaning that `CruiseControlOperation` has been created and is waiting for the `add_broker` task to be finished.
```

Comment:
I think it would be better:  has been created and is waiting for the add_broker task execution (or something similar :)